Packs/GoogleCloudLogging/Integrations/GoogleCloudLogging/GoogleCloudLogging_description.md

```diff
@@ -0,0 +1,17 @@
+## Google Cloud Logging Help
+### Scope
+We need to provide the below mentioned OAuth scope to execute the commands: https://www.googleapis.com/auth/cloud-platform.
+ 
+### Create a Service Account
+1. Go to the [Google documentation](https://developers.google.com/identity/protocols/oauth2/service-account#creatinganaccount) and follow the procedure in the Creating a Service Account section. After you create a service account, a Service Account Private Key file is downloaded. You will need this file when configuring an instance of the integration.
+2. Grant the mentioned OAuth scope https://www.googleapis.com/auth/logging.read to the Service Account to enable the Service Account to perform certain Google Cloud API commands.
```

Comment:
Add the following here: https://cloud.google.com/logging/docs/reference/v2/rest/v2/entries/list#authorization-scopes
Give the user all the possibilities